internal/bminventory/inventory.go

```diff
@@ -464,9 +464,16 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 	log := logutil.FromContext(ctx, b.log).WithField(ctxparams.ClusterId, id)
 	log.Infof("Register cluster: %s with id %s and params %+v", swag.StringValue(params.NewClusterParams.Name), id, params.NewClusterParams)
 	success := false
+	fullCluster := &common.Cluster{}
```

Comment:
I think this can just be `cluster` or maybe change the name to `finalCluster` or `resultingCluster`.

"Full" makes me thing of HA mode.